src/Dockerfile

```diff
@@ -42,6 +44,10 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 
 COPY --from=icalialabs/wkhtmltopdf:0.12.5-stretch /usr/local/bin/wkhtmltopdf /usr/local/bin/wkhtmltopdf
 
+
+# Install pdoc3 for generating Python documentation
+RUN pip3 install pdoc3
```

Comment:
What about locking the version that we do not get a random version each time we build the image?

Most of the other dependencies are already soft-locked since they are pulled with apt-get and ubuntu/debian tends to keep version stable.